### PR TITLE
feat: contributor guidelines in hindi updated

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
Resolves #[Add issue number here]

 Changes:

 Screenshots of the change:

#### PR Checklist

- [ ] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

File: contributor_docs/hi/contributor_guidelines.md [hunk 1]
```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
 
 ---
 
-# Pull requests
+# पुल रिक्वेस्ट
```
Comment (jaiakash) [inline contributor_docs/hi/contributor_guidelines.md:409]: Request can be in hindi

File: contributor_docs/hi/contributor_guidelines.md [hunk 2]
```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
...
 
-![A screenshot of Github Desktop after pushing code. In the left sidebar, it says "0 changed items." In the right pane, below the "No local changes" header, a blue "Review Pull Request" button has been marked up with a red circle.](../images/preview-pull-request.png)
+GitHub डेस्कटॉप से, हेडर में शाखाएँ बदलने के बटन के ठीक दाईं ओर आपके परिवर्तनों को GitHub पर भेजने के लिए एक बटन है। अपने परिवर्तनों को आगे बढ़ाने के लिए इस पर क्लिक करें।![परिवर्तन करने के बाद GitHub डेस्कटॉप का एक दृश्य। परिवर्तनों को ऑनलाइन पुश करने का बटन लाल रंग में दर्शाया गया है।](../images/publish-branch.png)
 
-From the terminal, run the following command:
+
+एक बार आपका कोड अपलोड हो जाने पर, यह एक बटन दिखाएगा जो आपको पुल रिक्वेस्ट बनाने के लिए प्रेरित करेगा। बटन पर एक बार क्लिक करने से वास्तव में रिक्वेस्ट बनाने के लिए दूसरे बटन के साथ पूर्वावलोकन दिखाई देगा। प्रक्रिया शुरू करने के लिए "क्रिएट पुल रिक्वेस्ट" बटन दबाएँ।
+
+![कोड पुश करने के बाद Github Desktop का स्क्रीनशॉट। बाएँ साइडबार में, यह लिखा है "0 changed items." दाएँ फलक में, "No local changes" शीर्षक के नीचे, एक नीला "Review Pull Request" बटन को लाल वृत्त के साथ चिह्नित किया गया है।](../images/preview-pull-request.png)
```
Comment (jaiakash) [inline contributor_docs/hi/contributor_guidelines.md:424]: Gtihub Desktop in hindi

File: contributor_docs/hi/contributor_guidelines.md [hunk 3]
```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
...
+यहां पहला कदम आपके नए कमिट को आपके p5.js के फोर्क पर धकेलना है; इसे अपने फोर्क में परिवर्तन अपलोड करने के रूप में सोचें।
 
-Once your code is uploaded, it will show a button prompting you to create a pull request. Clicking the button once will show a preview with another button to actually create the request. Press the "Create Pull Request" button to begin the process.
 
-![A screenshot of Github Desktop after pushing code. In the left sidebar, it says "0 changed items." In the right pane, below the "No local changes" header, a blue "Review Pull Request" button has been marked up with a red circle.](../images/preview-pull-request.png)
+GitHub डेस्कटॉप से, हेडर में शाखाएँ बदलने के बटन के ठीक दाईं ओर आपके परिवर्तनों को GitHub पर भेजने के लिए एक बटन है। अपने परिवर्तनों को आगे बढ़ाने के लिए इस पर क्लिक करें।![परिवर्तन करने के बाद GitHub डेस्कटॉप का एक दृश्य। परिवर्तनों को ऑनलाइन पुश करने का बटन लाल रंग में दर्शाया गया है।](../images/publish-branch.png)
```
Comment (jaiakash) [inline contributor_docs/hi/contributor_guidelines.md:419]: Github in hindi

File: contributor_docs/hi/contributor_guidelines.md [hunk 4]
```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
...
+![कोड पुश करने के बाद Github Desktop का स्क्रीनशॉट। बाएँ साइडबार में, यह लिखा है "0 changed items." दाएँ फलक में, "No local changes" शीर्षक के नीचे, एक नीला "Review Pull Request" बटन को लाल वृत्त के साथ चिह्नित किया गया है।](../images/preview-pull-request.png)
+
+टर्मिनल से, निम्नलिखित कमांड चलाएँ:
 
 ```
 git push -u origin [branch_name]
 ```
 
-Once the push is complete, you may see a link in the terminal that lets you open a pull request, if not you can navigate to your fork in your web browser, switch to the branch you are working on with the dropdown button on top of the file list, click on "Contribute" then "Open pull request."
+एक बार पुश पूरा हो जाने पर, आप टर्मिनल में एक लिंक देख सकते हैं जो आपको पुल रिक्वेस्ट खोलने की सुविधा देता है, यदि नहीं तो आप अपने वेब ब्राउज़र में अपने फोर्क पर नेविगेट कर सकते हैं, शीर्ष पर ड्रॉपडाउन बटन के साथ उस ब्रांच पर स्विच करें जिस पर आप काम कर रहे हैं फ़ाइल सूची में, "Contribute" पर क्लिक करें और फिर "Open pull request." पर क्लिक करें।
```
Comment (jaiakash) [inline contributor_docs/hi/contributor_guidelines.md:432]: pull request  and navigate in hindi

File: contributor_docs/hi/contributor_guidelines.md [hunk 5]
```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
...
 
 ![](https://lh7-us.googleusercontent.com/xoqM9gLSFXyw7b3gzG8JlHqoO0zxHALbjSz93E3D0HNh4Jw2wDWdzHKUEchnB6hdjQC7hVn-o5prrXkLw2WiEOoVKJF6kpmyA65sirN0z-Vy3PBY3rCXC5Ifn5stQhcdUQxQS0rsVoW0_hlkfTcY8PQ)
 
-You may also see a button to open a pull request when you visit the p5.js Github repo. Clicking it will also work to open a new pull request.
+जब आप p5.js Github रेपो पर जाते हैं तो आपको पुल रिक्वेस्ट खोलने के लिए एक बटन भी दिखाई दे सकता है। इसे क्लिक करने से एक नया पुल रिक्वेस्ट खोलने का भी काम किया जा सकता है।
+
+![p5.js GitHub रिपॉजिटरी वेब पेज के मुख्य पृष्ठ का क्रॉप किया हुआ स्क्रीनशॉट। पृष्ठ के शीर्ष के पास एक अनुभाग एक पीला कॉल टू एक्शन बॉक्स है जिसमें "Compare & pull request" टेक्स्ट वाला हरा बटन है।](../images/recent-pushes)
```
Comment (jaiakash) [inline contributor_docs/hi/contributor_guidelines.md:438]: crop, action box

File: contributor_docs/hi/contributor_guidelines.md (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:424]: I think we can keep it as GitHub Desktop as it is name of the application which hey will read in English by default.

File: contributor_docs/hi/contributor_guidelines.md (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:409]: it is left in English as the user will read the same English word at other place like while using GitHub I don't want them to get confused on it's hindi alternative.

**Pull request merged**